config/routes.rb

```diff
@@ -3,7 +3,13 @@
 
   # Defines the root path route ("/")
   root "users#index"
-  get "/users/:id", to: "users#show"
-  get "/users/:user_id/posts", to: "posts#index"
-  get "/users/:user_id/posts/:id", to: "posts#show"
+  # get "/users/:id", to: "users#show"
+  # get "/users/:user_id/posts", to: "posts#index"
+  # get "/users/:user_id/posts/:id", to: "posts#show"
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: